code/game/machinery/PDApainter.dm

```diff
@@ -103,6 +103,12 @@
 		stored_id_card = null
 		update_appearance(UPDATE_ICON)
 
+/obj/machinery/pdapainter/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	default_unfasten_wrench(user, tool)
+	power_change()
```

Comment:
Should keep the if check, it has other attached checks outside just "is this a wrench"